pkg/operator/vttablet/mysqlctld.go

```diff
@@ -47,6 +48,9 @@ for mycnf in $(find . -mindepth 2 -maxdepth 2 -path './vt_*/my.cnf'); do
   sed -i -e 's,^socket[ \t]*=.*$,socket = ` + mysqlSocketPath + `,' "${mycnf}"
 done
 `
+
+	defaultInitCPURequestMillis   = 100
+	defaultInitMemoryRequestBytes = 32 * (1 << 20) // 32 MiB
```

Comment:
Default values live here https://github.com/planetscale/vitess-operator/blob/main/pkg/apis/planetscale/v2/defaults.go, and so should these.